build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -47,6 +47,8 @@ def substituteFluentBitPlaceHolders
     ignoreOlder = ENV["FBIT_TAIL_IGNORE_OLDER"]
```

Comment:
Please add the agent resource usage charts with and without kubernetes metadata to the PR to validate and refer the resource usage consumption.

Reply:
included in the scale test doc